### PR TITLE
Feature/BISC-72 view registration backend

### PR DESCRIPTION
**Pull request by dirkkok**
This PR will:
- add view registration query
- expose memo and registrar fields or a registration (quite a lot of changes were required to make this work)

### CODE REVIEW

File: server/src/CommonGroundAPI/cc/OrganizationRepository.ts [hunk 1]
```diff
@@ -31,15 +31,16 @@ type OrganizationEntity = {
...
     email?: string
     emailId?: string
-    sourceOrganization: string
-    address: {
+    sourceOrganization?: string
+    address?: {
         id: string
         street: string
         houseNumber: string
         houseNumberSuffix: string
```
Comment (Jorann) [inline server/src/CommonGroundAPI/cc/OrganizationRepository.ts:39]: suffix should have a `?`

File: server/src/Student/StudentResolver.ts [hunk 2]
```diff
@@ -24,27 +25,6 @@ class FindAcceptAndDeleteRegistrationArgs {
...
-    @Field()
-    public givenName!: string
-
-    @Field()
-    public additionalName?: string
-
-    @Field()
-    public familyName!: string
-}
-
```
Comment (Jorann) [inline server/src/Student/StudentResolver.ts:47]: I love this cleanup

**Review by Jorann — CHANGES_REQUESTED** [review]
One small comment

File: server/src/CommonGroundAPI/cc/OrganizationRepository.ts (hunk 1 above)
Comment (dirkkok) [reply server/src/CommonGroundAPI/cc/OrganizationRepository.ts]: Fixed!

**Review by Jorann — APPROVED** [review]

**Pull request merged**